identity/Cargo.toml

```diff
@@ -5,8 +5,13 @@ authors = ["Markus Merklinger <markus.merklinger@code.berlin>"]
 edition = "2018"
 
 [lib]
-name = "identity_rpc_service"
-path = "src/rpc/service.rs"
+name = "identity"
```

Comment:
As we discussed in the past we should stick to some naming convention.
```rust
// either
use identity_lib;
// or
use identity;
```

I am actually more in favor of the option w/o the `_lib`-prefix. What do you say?

Reply:
As we already have the other pattern, I will adapt to it.

Reply:
Fixed.